### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:

- Read the Leaderboard API documentation and learn how to use it.
- Create a new game with the name of my choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+  fetch(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      user: nameValue.value,
+      score: scoreValue.value,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => response.json());
```
Comment (Mre55) [inline src/index.js:19]: [OPTIONAL] After this line you can clear the input field to allow the user to enter another value. You can do so by using something like this:
```
nameValue.value = '';
scoreValue.value = '';
```

File: src/modules/display_data.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/SiR96bAia8ODy3OSMi9e/scores';
+
+export const scoreUl = document.querySelector('.scores');
+
+export default async function displayData() {
```
Comment (Mre55) [inline src/modules/display_data.js]: Nice job your application is working as expected 👍. Let's convert this function into an `arrow function` you can refer to [here](https://www.digitalocean.com/community/tutorials/js-async-functions#async-arrow-function) for how to do so. Don't forget to export it as `const` and import it as `import { displayData, url, scoreUl } inside `index.js` file.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @ksigei,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._





## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by phelian23 — APPROVED** [review]
Hi @ksigei, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**